modules/gitops-configuring-gitops-managed-resources-to-use-mounted-secrets.adoc

```diff
@@ -0,0 +1,117 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-gitops-managed-resources-to-use-mounted-secrets_{context}"]
+= Configuring {gitops-shortname} managed resources to use mounted secrets
+
+You must configure the {gitops-shortname} managed resources by adding volume mounts configuration to a deployment and configure the container pod to use the mounted secret. For example, consider that you want to add volume mounts configuration to the deployment of `app-taxi` application and the `100-deployment.yaml` file is in the `/environments/dev/apps/app-taxi/services/taxi/base/config/` directory.
+
+.Prerequisites
+
+* You have the AWS Secrets Manager resources stored in your {gitops-shortname} repository.
+* You have the Secrets Store Container Storage Interface (SSCSI) driver configured to mount secrets from AWS Secrets Manager.
+
+.Procedure
+
+. Add volume mounts configuration in the target resource:
+
+.. Create a `100-deployment.yaml` file to use the secret provider class resources:
```

Comment:
`100-deployment.yaml` is just an example. User doesn't need to create a new resource just to use the secret. The point here is to add the volume mounting configuration to an existing resource. For example we can update an existing deployment file as below in order to mount the secret. Even no need to mention the file name, because it's too specific. You can refer my markdown file [here](https://github.com/ciiay/gitops-operator/blob/gitops-3394-integrate-sscsid-with-gitops/docs/Integrate%20GitOps%20with%20Secrets%20Management.md?plain=1#L311).

Reply:
Addressed.